public/components/monitoring/tests/use_monitoring.test.ts

```diff
@@ -124,6 +124,62 @@ describe('useMonitoring', () => {
     });
     await waitFor(() => expect(Model.prototype.search).toHaveBeenCalledTimes(2));
   });
+
+  it('should call searchByNameOrId with from 0 after page changed', async () => {
+    const { result, waitFor } = renderHook(() => useMonitoring());
+
+    result.current.handleTableChange({
+      pagination: {
+        currentPage: 2,
+        pageSize: 15,
+      },
+    });
+
+    await waitFor(() => {
+      expect(result.current.pagination?.currentPage).toBe(2);
+    });
+
+    act(() => {
```

Comment:
Seems it doesn't need `act()`

Reply:
Can be removed. Since we've already call `waitFor` below.